app/controller/command/commands/team.py

```diff
@@ -635,7 +639,8 @@ def refresh_helper(self, user_id) -> ResponseTuple:
         Ensure that the local team database is the same as GitHub's.
 
         In the event that our local team database is outdated compared to
-        the teams on GitHub, this command can be called to fix things.
+        the teams on GitHub, this command can be called to fix these
+        inconsistencies.
```

Comment:
This likely ties into https://github.com/ubclaunchpad/rocket2/issues/490 in that we can't remove this command, as we'll always need something to forcibly bring everything in line

Reply:
Probably need to add a `Deprecation Warning` whenever someone calls the command.

Reply:
Oh my point was I _dont_ think we can deprecate it, ever, since we need some way to force a state sync (for example, to add new features - essentially a "migration")